scripts/cloudfoundry/bat-cdn/cdn-config.yml

```diff
@@ -0,0 +1,23 @@
+headers: &headers
+  - Accept
+  - Authorization
+qa:
+  headers: *headers
+  domain:
+    qa.find-postgraduate-teacher-training.service.gov.uk
```

Comment:
This would give more flexibility in the script if this is stored as a list

Reply:
done...